### DISCUSSION TITLE
What do "*" and "_"  mean in aaSeqCDR3

### DISCUSSION THREAD

**Discussion by Masakazu-Ishikawa137** [milaboratory/mixcr#996] [Q&A] [answered]
Hi,

I'm using mixcr analyze, and I got outputs.
When I see the output, I found "CAV*GE_TPLVF" in aaSeqCDR3 column.

What does sequence mean?

**Comment by mizraelson** [comment, marked as answer]
Hi, this means the clone is non-functional, it can't produce a viable protein.
`_` - is an incomplete codon ( more on that here https://docs.milaboratories.com/mixcr/reference/ref-translation-rules/?h=translation )
In your case since `_` is inside CDR3 sequence it means that the number of N nucleotides resulted in an out-of-frame sequence.
`*` - is a Stop-codon

**Reply by Masakazu-Ishikawa137** [reply]
OK, I understood. Thank you so much!
